.gitignore

```diff
@@ -1,30 +1,30 @@
 # See https://help.github.com/ignore-files/ for more about ignoring files.
 
 # compiled output
-/dist/
-/tmp/
+dist/
+tmp/
 
 # dependencies
-/bower_components/
-/node_modules/
+bower_components/
+node_modules/
 
 # misc
-/.env*
-/.pnp*
-/.sass-cache
-/connect.lock
-/coverage/
-/libpeerconnection.log
-/npm-debug.log*
-/testem.log
-/yarn-error.log
+.env*
+.pnp*
+.sass-cache
+connect.lock
+coverage/
+libpeerconnection.log
+npm-debug.log*
+testem.log
+yarn-error.log
 
 # ember-try
```

Comment:
seem like all the below lines not needed any more and only copy what we have in `addon/.gitignore`?

Reply:
yeah, I'll address in a follow up